pageComponents/NewPostPage/HeroPositionSelect.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import Select, { Option } from "@@/components/Select";
+import { POSITION_CORRESPONDENCE_EACH_PLAYER_LENGTH } from "@@/utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
```

Comment:
I'm quite wondering that some form elements have `defaultValue` prop but some don't have. What kind of factor divides between them?

Reply:
And please don't forget about YAGNI. Less code is better.

Reply:
I meant some input components look controlled, and other components are uncontrolled. I'm still wondering what kind of reason you use to decide.